python/taulabs/telemetry.py

```diff
@@ -323,13 +323,19 @@ def _do_io(self, finish_time):
         if r:
             # Shouldn't throw an exception-- they just told us
             # it was ready for read.
-            chunk = os.read(self.fd, 1024)
-            if chunk == '':
-                raise RuntimeError("stream closed")
+            try:
```

Comment:
PS: I never saw it when parsing log files, only when connecting to the POSIX simulation using TCP